includes/Checker/Runtime_Environment_Setup.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Runtime_Environment_Setup
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+/**
+ * Class to setup the Runtime Environment for Runtime checks.
+ *
+ * @since n.e.x.t
+ */
+class Runtime_Environment_Setup {
+
+	/**
+	 * Sets up the WordPress environment for runtime checks
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return void
+	 */
+	public function setup() {
+		global $wpdb, $wp_filesystem;
+
+		require_once ABSPATH . '/wp-admin/includes/upgrade.php';
+
+		// Set the new prefix.
+		$prefix = $wpdb->set_prefix( 'wppc_' );
+
+		// Create and populate the test database tables if they do not exist.
+		if ( 'wppc_posts' !== $wpdb->get_var( "SHOW TABLES LIKE 'wppc_posts'" ) ) {
+			wp_install(
+				'Plugin Check',
+				'plugincheck',
+				'demo@plugincheck.test',
+				false
+			);
+		}
+
+		$wpdb->set_prefix( $prefix );
+
+		// Create the object-cache.php file.
+		if ( $wp_filesystem || WP_Filesystem() ) {
```

Comment:
Before doing this, we should do a check for `WP_PLUGIN_CHECK_PERFLAB_OBJECT_CACHE_DROPIN_VERSION` - if that constant is set (and set to whatever is the current version per the `object-cache.copy.php` file), we don't need to even try placing it since it would already be present.